bftengine/src/bftengine/messages/ViewChangeMsg.cpp

```diff
@@ -93,18 +98,49 @@ void ViewChangeMsg::addElement(SeqNum seqNum,
   b()->numberOfElements++;
 }
 
+void ViewChangeMsg::addComplaint(const ReplicaAsksToLeaveViewMsg* const complaint) {
+  // We store complaints in a size/value list
+  // Visual representation for List Of Complaints:
+  // +--------------------+-------------------------+--------------------+-------------------------+---
+  // |Size of next element|ReplicaAsksToLeaveViewMsg|Size of next element|ReplicaAsksToLeaveViewMsg|...
+  // +--------------------+-------------------------+--------------------+-------------------------+---
+
+  ConcordAssert(b()->numberOfComplaints > 0 || b()->sizeOfAllComplaints == 0);
+  size_t bodySize = b()->locationAfterLast;
```

Comment:
`auto bodySize` ?

Reply:
Yes, this will be better declared auto, I agree.